stage3-py.sh

```diff
@@ -39,8 +39,8 @@ pip install \
       jupyterlab_hdf \
       lsst-efd-client \
       jupyter_firefly_extensions \
-      lsst-rsp \
-      rsp-jupyter-extensions
+      git+https://github.com/lsst-sqre/lsst-rsp.git@tickets/DM-37661 \
+      git+https://github.com/lsst-sqre/rsp-jupyter-extensions.git@tickets/DM-37661
```

Comment:
Presumably you want to drop this change before you merge.

Reply:
Yes, I have that change queued up and ready to be pushed, but I don't want to do that until I know I won't need any more branch builds (which means having new PyPi packages for those two components).